### PR TITLE
Add "raw note" functionality to Author's Notes script

### PR DESCRIPTION
**Pull request by ioncorimenia**
This allows you to opt out of your note being put in brackets.

### CODE REVIEW

File: Resources-And-Guides/Scripts/AuthorsNote/shared.js [hunk 1]
```diff
@@ -4,4 +4,5 @@ if (!state.setup) {
   state.authorsNote = "" // String for Author's Note.
   state.authorsNoteDepth = 2 // AN inserted n lined from the end of context
   state.authorsNoteDisplay = true // Toggle display of the AN
+  state.rawAuthorsNote = false // Wether to not surround the note with [Author's note: ]
```
Comment (sink-chan) [inline Resources-And-Guides/Scripts/AuthorsNote/shared.js]: Wether -> Whether

File: Resources-And-Guides/Scripts/AuthorsNote/inputModifier.js [hunk 2]
```diff
@@ -28,7 +28,14 @@ const modifier = (text) => {
         && -1 === authorsNoteIndex
       ) {
         authorsNoteIndex = index
-        state.authorsNote = authorsNoteTokens[1]
+        if (authorsNoteTokens[1].startsWith("-r ")) {
+          state.rawAuthorsNote = true
+          state.authorsNote = authorsNoteTokens[1].substring(3)
```
Comment (sink-chan) [inline Resources-And-Guides/Scripts/AuthorsNote/inputModifier.js]: This should be fine, but we might want to consider the case when only "/an -r " is passed. I suppose in that case state.authorsNote would be set to ''.

File: Resources-And-Guides/Scripts/AuthorsNote/inputModifier.js [hunk 3]
```diff
@@ -28,7 +28,14 @@ const modifier = (text) => {
...
+        }
+        else {
+          state.rawAuthorsNote = false
```
Comment (sink-chan) [inline Resources-And-Guides/Scripts/AuthorsNote/inputModifier.js:41]: Thank you for having this in here.

**Comment by sink-chan** [comment]
Looked inside scenario.zip file too and everything appears to be in order!

There's a minor typographical error in the comment of the shared.js change and I want to test the script behavior on input "/an -r ". Other than that, it looks good! You can update the comment @ioncorimenia or I can later. I don't think the PR needs to be held up over this.

**Comment by ioncorimenia** [comment]
I was thinking, perhaps typing only `/an` without any arguments should clear the author's note and show "Author's note disabled" in the status line (if you have display turned on)? It would provide a convenient way of temporarily disabling the script. If that's not a bad idea, I can send it as a separate pull request.

File: Resources-And-Guides/Scripts/AuthorsNote/shared.js [hunk 4]
```diff
@@ -2,7 +2,7 @@
 if (!state.setup) {
   state.setup = true // Ensure this is only set once and never wiped.
   state.authorsNote = "" // String for Author's Note.
-  state.authorsNoteDepth = 2 // AN inserted n lined from the end of context
+  state.authorsNoteDepth = 2 // AN inserted n lines from the end of context
```
Comment (sink-chan) [inline Resources-And-Guides/Scripts/AuthorsNote/shared.js:5]: Thank you! :+1:

**Comment by sink-chan** [comment]
Everything looks good on my end! @ioncorimenia @CoomerDeveloper I'll be merging the PR soon

**Comment by sink-chan** [comment]
Alright, merged in @ioncorimenia thanks a bunch!
(Also thanks for fixing the typo I had left in the scripts :laughing:)

**Pull request merged**